src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -706,6 +706,19 @@
     <Field type="OldObjectGcRoot" name="root" label="GC Root" />
   </Event>
 
+  <Event name="NativeMemoryUsagePart" category="Java Virtual Machine, Memory" label="Component Native Memory Usage" description="Native memory usage for a component" stackTrace="false" thread="false"
```

Comment:
I found it odd that naming for native memory tracking “parts” aren’t named consistently.  “Category” might be more obvious. I don’t have a strong opinion, just thought I’d mention my observation.

Reply:
Yes, these names have change a few times during dev and that's why it is out of sync. It should certainly be consistent, but it is really hard to come up with a great name for this. Especially since the different "parts" are so different, compare "GC" and "Object Monitors". Using "Category" has been discussed and I'm not against it, as always naming is very hard.